packages/cli/src/cli.ts

```diff
@@ -246,16 +277,17 @@ If one or more SCRIPT is given, these files are loaded instead of the directory.
 If the file 'wot-servient.conf.json' exists, that configuration is applied.
 
 Options:
-  -v,  --version                   display node-wot version
-  -i,  --inspect[=[host:]port]     activate inspector on host:port (default: 127.0.0.1:9229)
-  -ib, --inspect-brk[=[host:]port] activate inspector on host:port and break at start of user script
-  -c,  --clientonly                do not start any servers
-                                   (enables multiple instances without port conflicts)
-  -cp,  --compiler <module>        load module as a compiler
-                                   (The module must export a create function which returns
+  -v,   --version                   display node-wot version
+  -i,   --inspect[=[host:]port]     activate inspector on host:port (default: 127.0.0.1:9229)
+  -ib,  --inspect-brk[=[host:]port] activate inspector on host:port and break at start of user script
+  -c,   --clientonly                do not start any servers
+                                    (enables multiple instances without port conflicts)
+  -cp,  --compiler <module>         load module as a compiler
+                                    (The module must export a create function which returns
                                     an object with a compile method)
-  -f,  --configfile <file>         load configuration from specified file
-  -h,  --help                      show this help
+  -f,   --configfile <file>         load configuration from specified file
+  -h,   --help                      show this help
+        --[protocol].port           specify the port to expose the server for a specific protocol
```

Comment:
Shall we add an example in the description like the following

`--http.port 8081`

At first it wasn't very clear to me how to pass the argument
I had to look at the regex 🙃

Reply:
Good point! I will add an example!

Reply:
Done ✔